src/Nri/Ui/ClickableText/V3.elm

```diff
@@ -373,7 +373,7 @@ and essentially make the anchor a disabled placeholder.
 
 _Caveat!_
 
-The styleguide example will NOT work correctly because of <https://github.com/elm/browser/issues/34>, which describes a problem where "a tags without href generate a navigation event".
+The component-catalog example will NOT work correctly because of <https://github.com/elm/browser/issues/34>, which describes a problem where "a tags without href generate a navigation event".
```

Comment:
```suggestion
The Component Catalog example will NOT work correctly because of <https://github.com/elm/browser/issues/34>, which describes a problem where "a tags without href generate a navigation event".
```